manifests/params.pp

```diff
@@ -112,7 +112,7 @@
       }
     }
     'Debian': {
-      if ($facts['os']['name'] == 'ubuntu' and $facts['os']['distro']['codename'] == 'xenial') {
+      if ($facts['os']['name'] == 'ubuntu' and $facts['os']['release']['major'] == '16.04') {
```

Comment:
Slightly surprised this works. I'd expect it to be case sensitive.
```suggestion
      if ($facts['os']['name'] == 'Ubuntu' and $facts['os']['release']['major'] == '16.04') {
```

Reply:
Maybe it doesn't and our CI doesn't catch it :smile:

Reply:
Puppet string comparison is case-insensitive:

https://puppet.com/docs/puppet/7/lang_expressions.html#comparison-equality